functional/sev/run.sh

```diff
@@ -71,7 +71,7 @@ calculate_measurement_and_add_to_kbs() {
   if [[ -z "${measurement}" ]]; then >&2 echo "Measurement is invalid"; return 1; fi
 
   # Get encryption key from docker image label
-  enc_key=$(esudo docker inspect quay.io/kata-containers/encrypted-image-tests:encrypted \
+  enc_key=$(esudo docker inspect quay.io/kata-containers/encrypted-image-tests:unencrypted \
```

Comment:
Do we actually need to get the encryption key, add it to the DB, if we're dealing with an unencrypted image?

I mean, can't we just get rid of these steps if the tests are *not* using an encrypted image?

Reply:
Yeah slightly confusing but this test is for an encrypted image. This docker pull doesn't affect the image that we actually pull inside the guest. We just download the image because it has the ssh key and the encryption key in the labels and we need those for the test.